core/src/main/scala/kafka/utils/ShutdownableThread.scala

```diff
@@ -57,15 +57,14 @@ abstract class ShutdownableThread(val name: String, val isInterruptible: Boolean
   def doWork(): Unit
 
   override def run(): Unit = {
-    info("Starting ")
+    info("Starting %s %d".format(name, this.getId))
```

Comment:
This change doesn't really belong here, I introduced while debugging test cases.